src/main/res/layout/loading_image_view.xml

```diff
@@ -7,7 +7,8 @@
         android:id="@+id/imageView"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_gravity="center"/>
+        android:layout_gravity="center"
+        android:contentDescription="@string/loading_image_view_description"/>
```

Comment:
Is this message helpful to a visually impaired person? https://stackoverflow.com/a/26120042 ?